CMakeLists.txt

```diff
@@ -228,18 +236,41 @@ if(DEFINED ENV{CONDA_BUILD_STATE})
 		set(PNG_LIBRARY ${PNG_LIBRARIES} CACHE PATH "")
 		set(JPEG_LIBRARY ${JPEG_LIBRARIES} CACHE PATH "")
 		set(ZLIB_LIBRARY ${ZLIB_LIBRARIES} CACHE PATH "")
+
 		set(GSL_LIBRARY ${GSL_LIBRARIES})
+		set(GSL_INCLUDE_PATH ${GSL_INCLUDE_DIRS})
+		set(GSL_CBLAS_LIBRARY ${GSL_CLBAS_LIBRARY})
```

Comment:
Well, actually, looks correct according to cmake documentation, https://cmake.org/cmake/help/v3.2/module/FindGSL.html#cache-variables. This line makes sense only when `find_package(GSL REQUIRED)` is called, but I am leaving it. May come back and use `find_package(GSL REQUIRED)` in the future.